server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/ObjectStorageBlobsDAOTest.java

```diff
@@ -201,5 +203,13 @@ void readBytesShouldNotCompleteWhenDoesNotAwait() {
         Mono<byte[]> resultFuture = testee.readBytes(testee.getDefaultBucketName(), blobId).subscribeOn(Schedulers.elastic());
         assertThat(resultFuture.toFuture()).isNotCompleted();
     }
+
+    @Test
+    void lazilyCreateBucketWhenSaveCannotWork() {
```

Comment:
for tests that are not working:
 - write the expected behaviour
 - @Disable it

This will make clear this is not the intended behaviour.